en/includes/guides/account-configurations/user-onboarding/self-registration.md

```diff
@@ -173,3 +173,200 @@ Note the following details in the response payload:
 
 Once you have identified the `emailVerified` status of the user and the method by which the user is onboarded (self-registration or onboarded by an administrator), you can enforce any access restrictions for that user through your application logic.
 
+{% if product_name == "Asgardeo" or (product_name == "WSO2 Identity Server" and is_version > "7.1.0" ) %}
+## Resend account verification email
+
+Resending the account verification email helps users who didn't receive the initial email, deleted it, or found the verification link expired. This ensures users can complete the verification process and gain appropriate access to the application.
+
+!!! tip
+
+    To implement this scenario, you must enforce account verification for self-registered users and activate accounts immediately without waiting for verification.</br></br> Learn more in [enable/disable self-registration](#enabledisable-self-registration).
+
+- To resend the account verification email for your own account, use the resend-code API as shown below.
```

Comment:
shall we link to the resend code API doc?

Reply:
```suggestion
- To resend the account verification email for the authenticated user, use the resend-code API as shown below.
```

Reply:
> shall we link to the resend code API doc?

In asgardeo we don't have exposed this API

Reply:
Resolved via https://github.com/wso2/docs-is/pull/5355/commits/8aab8b464237b6ab3565afe5ecc80b52e7d3197a